docs/howtos/ensembling.rst

```diff
@@ -0,0 +1,25 @@
+Ensembling on multiple devices
+=============================
+
+We show you how to rewrite code for training a single mode to training an 
+ensemble of models, one on each device.
+
+Example of the new directive:
+
+.. codediff::
+  :title_right: Wit ensembling
+  :highlight_left: 1
```

Comment:
Should we also consider a version where the line numbers aren't specified here, but instead embedded in the code via a comment like `#  <highlight>`? (that would then get stripped in the output view, and instead that line would be highlighted)

The upside is that it's easier to then edit HOWTOs without having to recompute the line numbers...

WDYT?

Reply:
Yes I think that is actually nicer. I've implemented it and also simplified the code a bit.